### PR TITLE
Deployment tests

### PR DESCRIPTION
**Pull request by elvinm**
# Description and Motivation 
Adding integration tests for creation of deployments in edge crawl scenario.

...

# Change type 

- [ ] Bug fix
- [x] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: Helper/Constants.cs [hunk 1]
```diff
@@ -5,12 +5,12 @@ namespace Helpers
     public class Constants
     {
         // Endpoints
-        public const string CONFIG_ADDRESS = "http://127.0.0.1:9005/v1";
+        public const string CONFIG_ADDRESS = "http://localhost:9005/v1";
         public const string TELEMETRY_ADDRESS = "http://127.0.0.1:9004/v1";
```
Comment (isaac73) [inline Helper/Constants.cs]: Can you please convert all the addresses from 127.0.0.1 to localhost?

File: IoTHubManager/DeploymentsTests.cs [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+using IoTHubManager.Models;
+using Xunit;
+using Newtonsoft.Json;
+
+namespace IoTHubManager
+{
+    [Collection("IoTHub Manager Tests")]
+    public class DeploymentsTests
+    {
+        private const string TEST_PACKAGE_JSON =
```
Comment (isaac73) [inline IoTHubManager/DeploymentsTests.cs]: Can you copy this into Constants file or some common file. I see this is also being used in packageTest

File: IoTHubManager/DeploymentsTests.cs [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+                @"{
+                    ""id"": ""tempid"",
+                    ""schemaVersion"": ""1.0"",
+                    ""content"": {
```
Comment (isaac73) [inline IoTHubManager/DeploymentsTests.cs]: The escape chars in this Json looks weird. Is this test passing for both Java and Dotnet containers?

**Review by isaac-dasan — CHANGES_REQUESTED** [review]

File: Helper/Constants.cs (hunk 1 above)
Comment (elvinm) [reply Helper/Constants.cs]: Updated all to localhost. Need to make sure this still works fine in the test builds but it should.

File: IoTHubManager/DeploymentsTests.cs (hunk 2 above)
Comment (elvinm) [reply IoTHubManager/DeploymentsTests.cs]: Done but updating in https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet-tests/pull/42 because I pressed squash & merge.

File: IoTHubManager/DeploymentsTests.cs (hunk 3 above)
Comment (elvinm) [reply IoTHubManager/DeploymentsTests.cs]: Yes I had tested this with both java and .net. It is just the syntax of C# verbatim strings.
